src/chunkserver/chunkserver-common/default_disk_manager.h

```diff
@@ -47,6 +47,11 @@ class DefaultDiskManager : public IDiskManager {
 	/// Virtual destructor needed for correct polymorphism
 	virtual ~DefaultDiskManager() = default;
 
+	virtual int parseCfgLine(std::string hddCfgLine);
+
```

Comment:
nit: If DefaultDiskManager only uses this, maybe it should be private?

Reply:
The derived classes will reimplement it. The unit tests also will use it.